qtconsole/rich_jupyter_widget.py

```diff
@@ -140,9 +140,8 @@ def _handle_execute_result(self, msg):
                 self._pre_image_append(msg, prompt_number)
                 try:
                     self._append_latex(data['text/latex'], True)
-                except Exception:
-                    self.log.error("Failed to render latex: '%s'", data['text/latex'], exc_info=True)
```

Comment:
good points. it’s the latex-repr-writers’ responsibility to emit correct code anyway and if they don’t, they’ll have to debug it.